pkg/server/metrics.go

```diff
@@ -0,0 +1,52 @@
+package server
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/google/go-github/github"
+)
+
+var (
+	rateLimit = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "github_rate_limits",
+		Help: "Rate limit for GitHub API calls",
+	}, []string{"type"})
+
+	webhookCounter = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Name: "webhook_counter",
+		Help: "A counter of the webhooks made to service.",
+	}, []string{"event_type"})
+
+	handledEventsCounter = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Name: "handled_events_counter",
+		Help: "A counter of handled events.",
+	}, []string{"event_type"})
+)
+
+func init() {
+	prometheus.MustRegister(rateLimit)
```

Comment:
please use `prometheus.register` and handle the error properly

Reply:
@MatousJobanek I believe in an ideal scenario in any service, Prometheus should register all required collector. but if something bad happens shouldn't be it panic?

In face Kubernetes project is also using this same way https://github.com/kubernetes/kubernetes/blob/master/pkg/proxy/metrics/metrics.go#L45